doc/index.rst

```diff
@@ -2,14 +2,11 @@
 
 .. title:: Matplotlib: Python plotting
 
-Matplotlib is a Python 2D plotting library which produces publication quality
-figures in a variety of hardcopy formats and interactive environments across
-platforms.  Matplotlib can be used in Python scripts, the Python and IPython_
-shells, the Jupyter_ notebook, web application servers, and four graphical user
-interface toolkits.
+Matplotlib: 2D visualization in Python
```

Comment:
I know 2D is in the original copy, but technically mpl supports enough 3D and there's enough really cool stuff done in 3D that I don't know that 2d should be emphasized here

Reply:
ok